Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -615,6 +617,33 @@ - (nullable NSString *)currentScreenNameForSessionReplay
                .firstObject;
 }
 
+- (void)showMaskPreview:(CGFloat)opacity
+{
+    if ([SentryDependencyContainer.sharedInstance.crashWrapper isBeingTraced] == NO) {
+        return;
+    }
+
+    UIWindow *window = SentryDependencyContainer.sharedInstance.application.windows.firstObject;
+    if (window == nil) {
+        SENTRY_LOG_WARN(@"There is no UIWindow available to display preview");
+        return;
+    }
+
+    if (_previewView == nil) {
+        _previewView = [[SentryMaskingPreviewView alloc] initWithRedactOptions:_replayOptions];
+    }
+
+    _previewView.opacity = opacity;
+    [_previewView setFrame:window.bounds];
+    [window addSubview:_previewView];
```

Comment:
`h`: What happens if a user calls this method multiple times in a row? If I'm not mistaken, the code adds multiple `_previewView`.

Reply:
Since it's the same instance (see lines 632 - 634), it just reads the same view to the main window. The window may have changed if its an iPad app.